addons/auth_totp_mail/models/res_users.py

```diff
@@ -199,11 +199,9 @@ def _send_totp_mail_code(self):
             'partner_ids': [],
             'scheduled_date': False,
         }
-        with self.env.cr.savepoint()
-            # C'est jamais dans la queue, ou bien on rollack et c'est bon
-            template.with_context(**context).send_mail(
-                self.id, force_send=True, raise_exception=True, email_values=email_values, email_layout_xmlid='mail.mail_notification_light'
-            )
+        template.with_context(**context).send_mail(
```

Comment:
If the template is broken (not only a queue issue), are you going to break the whole flow ?

Reply:
Moved to #198810